test/controllers/graphql_controller_7_test.rb

```diff
@@ -452,6 +452,16 @@ def teardown
     assert_equal [pm.id], JSON.parse(@response.body)['data']['project_medias']['edges'].collect{ |x| x['node']['dbid'] }
   end
 
+  test "should update task files" do
+    u = create_user is_admin: true
+    t = create_task
+    f = Rack::Test::UploadedFile.new(File.join(Rails.root, 'test', 'data', 'rails.png'), 'image/png')
+    authenticate_with_user(u)
+    query = 'mutation { updateTask(input: { clientMutationId: "1", id: "' + t.graphql_id + '" }) { task { id } } }'
+    post :create, params: { query: query, team: t.slug, file: { '0' => f } }
+    assert_response :success
+  end
+
```

Comment:
@caiosba when you have some time:
I know we talked about this yesterday, but why is this specific test needed? I'm not sure, but I think we talked about the uploading of files in the tipline, there was nothing that was needed regarding that, or are they related?

Reply:
@vasconsaurus , that part of the code is about uploading multiple files when calling a mutation, so the mutation itself doesn't really matter... since the test is about uploading files, I used a simpler mutation. In practice, yes, we use it in the application for tipline images and annotations.

Reply:
gracias ❤️